components/avatar/utils.js

```diff
@@ -0,0 +1,9 @@
+export const extractInitialsFromName = (fullName) => {
+  if (typeof fullName !== 'string' || !fullName.trim()) return '';
+
+  const names = fullName.trim().split(' ');
+
+  return names.length === 1
+    ? names.join('').substring(0, 2)
+    : names.slice(0, 2).map(name => name.substring(0, 1).toUpperCase()).join('');
```

Comment:
Why do we `slice(0, 2)` what if the user had like 6 names? We should always use the first and the last.

Reply:
That makes sense, I was thinking on just a name and a last name, fixing